### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by dd-obua**
### In this project I have:
* Generated cards in the works section dynamically using JavaScript objects and
* Implemented popup windows for both mobile and desktop versions

### CODE REVIEW

File: js/popup.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+const cancelMobilePopup = document.querySelector('.close-mobile-popup');
+const cancelDesktopPopup = document.querySelector('.close-desktop-popup');
+
+cancelMobilePopup.addEventListener('click', () => {
+  mobilePopup.classList.add('take-out');
+});
+
+cancelDesktopPopup.closest('.img-div').addEventListener('click', () => {
+  desktopPopup.classList.add('take-out');
+});
```
Comment (rotimiazeez) [inline js/popup.js]: Good job so far 👏🏽. Please note that in the original design, when the pop-up appears the background is blurred but that is not the case with your design. Kindly fix.
Your Design
Screenshot from 2023-07-29 15-19-39
Figma design
Screenshot from 2023-07-29 15-12-54

Comment (rotimiazeez) [reply js/popup.js]: Please note that on the mobile screen when I click the see more button, the pop-up appears below the page 😢. Kindly fix this. 👍🏽
[Screencast from 29-07-2023 15:10:35.webm](https://github.com/dd-obua/Mic-Portfolio/assets/44624138/f88353f8-27ec-4fe7-93d0-77a0bfd113f9)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by mikethreels — APPROVED** [review]
Hi Dan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**